### ISSUE TITLE
Weekend for weeks starting on Monday

### ISSUE THREAD

**Issue by igorsantos07** [Templarian/MaterialDesign#7060]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://pictogrammers.com/library/mdi) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](https://pictogrammers.com/docs/library/mdi/guides/brand-icons/) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

Throughout the world there's a mixed preference for Sundays and Mondays being the first day of the week ([wikipedia map](https://en.wikipedia.org/wiki/Week#/media/File:First_Day_of_Week_World_Map.svg)). Thus, the current `calendar-weekend` icon is not inclusive, as it forces the user to understand weekend as a split between start and end of the week. That's usual only in most of Americas, South Asia, South Africa and some other countries, but in Europe, the rest of Asia (including China), parts of Oceania and South America, weeks start at Monday.

There are some countries where the week starts on Saturday, but most of those places can go well with the `calendar-week-begin`, as seen in this [weekend holiday map](https://i.redd.it/ent5hyry9dr11.png). 

The original #4197 proposed lots of variations, but only the split version was included (without explicitly saying it's a split version and not the single weekend version on the world). I propose a second version, `calendar-weekend-end` or `calendar-weekend-monday`, to be included, respecting the rest of world who doesn't understand why the week would start by its end :))

### Give us an example of what this icon might look like.

These were taken from the original issue, and could very well be a mirror of `calendar-week-begin`.

![59518257-99b52d80-8ec5-11e9-9087-dcb825998979](https://user-images.githubusercontent.com/532299/229935651-2ed74091-2aca-4da8-9610-72e52a3147c7.png)

**Comment by MrGrigri** [comment]
We already have a `calendar-weekend`. But we could do `calendar-week-end`.

![image](https://user-images.githubusercontent.com/4528551/230446284-9bb5080f-d0fd-4357-9a30-5aaf02dc4538.png)

**Comment by Simran-B** [comment]
week-end and weekend would cause naming conflicts for some packages. Needs to be a different name

**Comment by igorsantos07** [comment]
Indeed. It's sad the original request wasn't respected, as the current icon would make more sense as `weekend-split`.

Maybe this new one could be `weekend-end` and the split one aliased to `weekend-split`? Wouldn't solve the split-centric naming, but at least helps with understanding.

**Comment by mririgoyen** [comment]
This is an interesting case because it is a language conflict, but it is one we definitely need  to account for. I've marked this as breaking because we're going to make some changes at the v8 release.

- We are going to rename our current `calendar-weekend` to the plural `calendar-weekends`.
- We will add this new variation, and the opposite of `calendar-week-begin`, as `calendar-week-end`.
- We will also add the outlined version of `calendar-week-end`.
